lxl-web/src/lib/components/find/Filters.svelte

```diff
@@ -46,7 +46,9 @@
 				type="search"
 			/>
 			<BiSearch class="text-icon absolute top-3 left-2.5 text-sm" />
-			<MyLibrariesFilter />
+			{#if page.url.pathname === '/find'}
```

Comment:
does not work for `/en/find`

Reply:
Whoops! will fix

Reply:
fixed in https://github.com/libris/lxlviewer/pull/1283/commits/8a70b35f972ab5899f1f7f771a9fca2835a51bfd